.gitignore

```diff
@@ -276,3 +276,6 @@ build
 **/__snapshots__
 .env*
 !.env
+
+#local history folder
+/.history
```

Comment:
What in the IDE is happening here?

Reply:
Lol, its vs code local history folder i ignored.